src/types/commands.ts

```diff
@@ -1097,6 +1097,7 @@ export const enum CommandResult {
   Readonly = "Readonly",
   InvalidViewportSize = "InvalidViewportSize",
   InvalidScrollingDirection = "InvalidScrollingDirection",
+  NoViewportScroll = "NoViewportScroll",
```

Comment:
`ViewportScrollLimitsReached` or `ViewportWontScrool` ? But, once again, it's a total nitpick and maybe your name is the best one :)

Reply:
ViewportScrollLimitsReached sounds way better, I didn't have any good idea when coding. Thanks ! ;)